pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -72,6 +75,64 @@ func TestApplyIndexerCluster(t *testing.T) {
 	splunkDeletionTester(t, revised, deleteFunc)
 }
 
+func TestApplyIndexerClusterMultipart(t *testing.T) {
```

Comment:
Maybe we can have another test just to test creation of CM part with `Replicas:0` ?

Reply:
Thanks for your feedback :-) Added unit test TestApplyIndexerClusterMasterOnly